includes/razorpay-webhook.php

```diff
@@ -331,6 +343,19 @@ protected function getPaymentEntity($razorpayPaymentId, $data)
         return $payment;
     }
 
+    /**
+     * Returns boolean false incase not proper webhook data
+     */
+    protected function shouldConsumeWebhook($data)
+    { 
+        if (isset($data['payload']['payment']['entity']['notes']['woocommerce_order_number']) === true)
```

Comment:
@ChetanGN , let add events as well in this list with OR condition and use it once only, not with every event.